docs/contributing.rst

```diff
@@ -199,12 +199,7 @@ A checklist for creating, testing, and distributing a new version.
 
 #. View your tag: ``git tag -v {number}``
 #. Push your tag: ``git push upstream {number}``.
-#. Delete old distributions: ``rm dist/*``.
-#. Create distributions with ``python setup.py sdist bdist_wheel``.
-#. Set your TestPyPI and canon PyPI credentials in your session with
-   ``keyring`` (docs forthcoming).
-#. Upload to Test PyPI: :command:`twine upload --repository-url
-   https://test.pypi.org/legacy/ --skip-existing dist/*`
+#. Upload to TestPyPI with ``TWINE_REPOSITORY=https://test.pypi.org/legacy/ tox -e release``
```

Comment:
Tiny detail, mostly bikeshedding: what do you think about inverting this to so that `tox -e release` defaults to Test PyPI, and requires an override for PyPI?

I did that using as a safety net in a [recent project](https://github.com/EnergySage/pytest-quarantine/blob/115ca9d1a9e8c5da8ecbbe3a6a005a166430c4bb/tox.ini#L57) (using `posargs` instead of the environment variable, because I didn’t know about the latter 😉)

Reply:
If changing the default behavior here, that suggests I should change it for the dozens of other projects I maintain and suggests that the default behavior for twine (uploading to pypi) is incorrect... and would demand that every project explicitly mention the default index. It seems to me that if it's best practice to default to the test index and to require an explicit override to upload to the production index that twine should implement that behavior and not just twine's release workflow. I'm hoping to eliminate the "upload to test" as part of the instructions anyway, to make that part of an automated release process. How about we keep it simple (more aligned with default) for now and revisit later when we have CI-driven automatic releases?

Reply:
Sounds reasonable. Thanks for the background.